### PR TITLE
Privacy page

### PR DESCRIPTION
**Pull request by Oxalisviolacea**
Description:  
* Add route, controller, and view for privacy page

Fixes #

* [x] All tests passing
* [x] Lint code
* [x] Checked local host
* [ ] Updated README

### CODE REVIEW

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
+  <a><%= link_to 'The Team', '/team' %></a>
   <a><%= link_to 'Contact' %></a>
   <a><%= link_to 'Help' %></a>
   <%= link_to 'Privacy Policy', "/privacy" %>
 </div>
 
 <div class="body">
-
+<br>
+<p>Moving to a new place is as exciting as it is stressful. Relocate was developed to help alleviate the stress so you
```
Comment (Callbritton) [inline app/views/home/index.html.erb:12]: This is great! I was reading back through it and thought maybe a minor revision for the first sentence could be "Moving to a new place can be exciting, but also stressful."

File: app/views/privacy/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (Callbritton) [inline app/views/privacy/index.html.erb:1]: Perfect!

File: app/views/team/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<h3>Overview</h3>
```
Comment (Callbritton) [inline app/views/team/index.html.erb:1]: Really wonderful job with this, team! Thank you!

File: app/views/utilities/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<h1>Local Utilites</h1>
+
```
Comment (Callbritton) [inline app/views/utilities/index.html.erb:2]: We will be expanding this to have each utility's name to be a link to their show page, right?

**Review by Callbritton — APPROVED** [review]
Everything looks good to me! I added one comment with a recommendation for a slight wording change, but this should be good to go!

File: app/views/utilities/index.html.erb (hunk 4 above)
Comment (dcoleman21) [reply app/views/utilities/index.html.erb:2]: Yes, I think that we chatted about that being our way forward.

**Review by dcoleman21 — APPROVED** [review]
Looks good. Let's just change that one sentence per Chris' suggestion.

File: app/views/home/index.html.erb (hunk 1 above)
Comment (Oxalisviolacea) [reply app/views/home/index.html.erb:12]: Great, I'll change that. Thanks!

**Pull request merged**